### PR TITLE
Update lime-example and move it into lime-docs

### PR DESCRIPTION
**Pull request by ilario**

### CODE REVIEW

**Comment by ilario** [comment]
Do you think I should move adhoc options in the main wifi configuration of lime-example or leave them in the specific one? Related discussion on #174

**Comment by ilario** [comment]
There are conflicts caused by merge of #176, I will solve them in a while.

File: packages/lime-debug/Makefile [hunk 1]
```diff
@@ -23,8 +23,9 @@ define Package/$(PKG_NAME)
 	MAINTAINER:=Gioacchino Mazzurco <gio@eigenlab.org>
 	URL:=http://libremesh.org
 	DEPENDS:=+PACKAGE_lime-proto-batadv:batctl \
-		+busybox +ethtool +iwinfo +iw +mtr +ip +iputils-ping6 +iputils-ping \
-		+sprunge +safe-reboot +netperf +pv +tcpdump-mini +bwm-ng
+       		+busybox +ethtool +iwinfo +iw +mtr +ip +iputils-ping6 +iputils-ping \
+		+sprunge +safe-reboot +netperf +pv +tcpdump-mini \
+		+bwm-ng
```
Comment (nicopace) [inline packages/lime-debug/Makefile]: This change shouldn't be here, as it is just aesthetic

File: packages/lime-proto-bmx6/Makefile [hunk 2]
```diff
@@ -7,7 +7,7 @@
 include $(TOPDIR)/rules.mk
 
 LIME_BUILDDATE:=$(shell date +%Y%m%d_%H%M)
-LIME_CODENAME:=bigbang
+LIME_CODENAME:=DaybootRelay
```
Comment (nicopace) [inline packages/lime-proto-bmx6/Makefile]: this change has nothing to do with the topic of documentation, neither changes related with this package

**Comment by ilario** [comment]
> There are conflicts caused by merge of #176, I will solve them in a while.

This. @nicopace ignore this PR until when I fix the things please.

File: packages/lime-proto-bmx6/Makefile (hunk 2 above)
Comment (ilario) [inline packages/lime-proto-bmx6/Makefile]: @nicopace ignore this PR until when I fix the things please.

File: packages/lime-debug/Makefile (hunk 1 above)
Comment (ilario) [inline packages/lime-debug/Makefile]: This doesn't come from me, it's from the revert of a Pau's commit 3e3c0f8 "Move lime-example", if you think it's better, I can avoid reverting it and build my PR on top of it.

**Comment by nicopace** [comment]
@ilario this is making builds break. can you check and correct ?

**Comment by ilario** [comment]
Ok, now it's clean.
lime-example still misses the specification of network for client mode, see discussion on #47 but this can be part of a successive PR.

**Pull request merged**